superset/assets/js_build.sh

```diff
@@ -20,6 +20,7 @@ cd "$(dirname "$0")"
 npm --version
 node --version
 time npm ci
-time npm run lint
-time npm run cover  # this also runs the tests, so no need to 'npm run test'
+#time npm run lint
+echo    "npm run test"
```

Comment:
Remove echo statement

Reply:
I don't see a command to invoke this file.
Can we segregate commands as per stages and remove empty stage

Reply:
Removed echo statement and and empty stage also.